apps/extension/src/state/wallets.ts

```diff
@@ -86,9 +85,6 @@ export const accountAddrSelector = (state: AllSlices) => (index: number, ephemer
 
   return {
     address: bech32Addr,
-    preview: ephemeral
```

Comment:
`preview` is no longer needed, as addresses are now shortened inside the `<Address />` component